test/ReactWrapper-spec.js

```diff
@@ -372,7 +372,7 @@ describeWithDOM('mount', () => {
 
     });
 
-    it('should not find components with invalid attributes', () => {
+    it.skip('should not find components with invalid attributes', () => {
```

Comment:
We could probably just use `sinon.spy` to mock `console.warn` in the test itself, and revert the mock (does sinon do that by itself? I'm not sure...) in the `beforeEach` block.